enacts/wat_bal/maproom_monit.py

```diff
@@ -372,15 +372,12 @@ def wat_bal_ts(
     return ts
 
 
-def plot_scatter(ts, name, color, dash=None, customdata=None):
-    hovertemplate = "%{y} on %{x}"
-    if customdata is not None:
-        hovertemplate = hovertemplate + " %{customdata}"
+def plot_scatter(ts, name, color, dash=None):
     return pgo.Scatter(
-        x=ts["T"].dt.strftime("%-d %b"),
+        x=ts["T"].data,
         y=ts.values,
-        customdata=customdata,
-        hovertemplate=hovertemplate,
+        #xaxis_tickformat="%d %B",
```

Comment:
The commit comment says this is broken, possibly because of missing values. I don't understand how there could  be missing values in T. What are the symptoms?

Reply:
you're right, it can't be missing values in T... Below is the error that is raised. All works fine when xaxis_tickformat is defined when using update_layout though.

ValueError: 'NoneType' object is not subscriptable

Invalid value received for the 'xaxis' property of scatter

    The 'xaxis' property is an identifier of a particular
    subplot, of type 'x', that may be specified as the string 'x'
    optionally followed by an integer >= 1
    (e.g. 'x', 'x1', 'x2', 'x3', etc.)
        

Property does not support subscripting:
xaxis_tickformat
^^^^^

Reply:
This is resolved, right?